foundations/html_css/html-foundations/elements-and-tags.md

```diff
@@ -23,7 +23,7 @@ A full paragraph element looks like this:
 
 You can think of elements as containers for content. The opening and closing tags tell the browser what content the element contains. The browser can then use that information to determine how it should interpret and format the content.
 
-There are some HTML elements that do not have a closing tag. These elements often look like this: `<br />` or `<img/>`, but some can also be used without the closing forward slash such as `<br>` or `<img>`. These are known as self-closing tags or empty elements because they don't wrap any content. We will encounter a few of these in later lessons, but for the most part, elements will have both opening and closing tags.
+There are some HTML elements that do not have a closing tag. These elements often look like this: `<br>` or `<img>`, but some can also be used without the closing forward slash such as `<br />` or `<img/>`. These are known as self-closing tags or empty elements because they don't wrap any content. We will encounter a few of these in later lessons, but for the most part, elements will have both opening and closing tags.
```

Comment:
I think the original is correct, but I might be misunderstanding. 

```
...be used without the closing forward slash such as `<br />`
```
The example is contradicting the text in the context of [self closing tags](https://teamtreehouse.com/community/closing-image-tag-img-or-img)